modules/va_facilities/spec/requests/facilities_request_spec.rb

```diff
@@ -370,4 +370,35 @@ def query_params(url)
       expect(response).to have_http_status(:unprocessable_entity)
     end
   end
+
+  context 'mobile flag' do
+    it 'responds with a boolean mobile flag for VHA facilities' do
+      create :vha_648A4
+      get base_query_path + '/vha_648A4', params: nil, headers: accept_json
+      expect(response).to be_successful
+      expect(response.body).to be_a(String)
+      json = JSON.parse(response.body)
+      expect(json['data']['attributes']['mobile']).to eq(false)
```

Comment:
If you want to check if the key *exists*, maybe `expect(json['data']['attributes']).to have_key('mobile')`

This is checking for a value which doesn't seem to be important for the spec.

Reply:
Considering that the value can be nil and shouldn't be nil for vha facilities, I think it's important to check the value here. But it does seem important to check for the key as well.

Reply:
@edmkitty awesome -- then the 2nd check is would be that its' not nil. That describes the context a lot better (it exists, is not nil) -- this implies false is a somehow expected.

Reply:
This is a good point. I like that method better. 👍